### ISSUE TITLE
[BITV/A11Y] Elements should not have tabindex greater then zeroCannot control

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/files_pdfviewer#635]

**Comment by danxuliu** [comment]
[`tabindex` was fixed in PDF.js 4.7.76](https://github.com/mozilla/pdf.js/commit/6f40ed729bf8b504bb47b1e28c4c26fb4601435f), so an update to that or a later version should fix this in the PDF viewer.
